lib/pure/options.nim

```diff
@@ -39,6 +39,29 @@ runnableExamples:
 ## inherits from `system.Defect` and should therefore never be caught.
 ## Instead, rely on checking if the option contains a value with the
 ## `isSome <#isSome,Option[T]>`_ and `isNone <#isNone,Option[T]>`_ procs.
+##
+##
+## Pattern matching
+## ================
+##
+## **Note:** This requires the [fusion](https://github.com/nim-lang/fusion) package.
```

Comment:
```suggestion
## .. Note:: This requires the [fusion](https://github.com/nim-lang/fusion) package.
```

For the new "Admonitions" thingy ?.

Reply:
Hmm, I haven't seen this yet, is it recommended to use it? If so, there should probably also be a PR to change this for all the other files.

Reply:
https://nim-lang.github.io/Nim/rst.html#features

Reply:
it looks good IMO:

![image](https://user-images.githubusercontent.com/2194784/108608774-1a584c80-737e-11eb-8171-8c71385dcdd0.png)